lib-view/src/routes/users/UserDialog.svelte

```diff
@@ -24,8 +24,8 @@
 			forename = user.forename;
 			surname = user.surname;
 			account = user.account;
-			role = user.role;
-			may_borrow = user.may_borrow;
+			role = user.role || '';
+			may_borrow = user.may_borrow || true;
```

Comment:
Yeah that is rather painfull and should be best solved in the api generation. You just should not have to know here that an undefined or null may_borrow is defaulted to true...